### DISCUSSION TITLE
HMR Support / Docs

### DISCUSSION THREAD

**Discussion by entity** [inertiajs/inertia#760] [Ideas]
It would be awesome if we can get a full guide on integrating HMR with Browser Sync, as currently no documentation for such.

**Comment by reinink** [comment]
You can already use HMR with Inertia. If you're using Laravel Mix, you just run `npm run hot`. This is more of a build tool thing than an Inertia thing.

**Reply by entity** [reply]
Thank you, I may be a noob however when I save it doesn’t refresh the page automatically? Is it 'HMR' that will automatically refresh my browser, but keep my state?

**Reply by reinink** [reply]
I'm honestly not sure, this would be a better question for the maintainers of Laravel Mix or Webpack. 👍

**Reply by entity** [reply]
> I'm honestly not sure, this would be a better question for the maintainers of Laravel Mix or Webpack. 👍

Thank you, wanted to query this with you as Vue has HMR which will automatically refresh the page upon save, but keep the person on the page with the components updated (and keep state).
